.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
No comments in json files
---
NB : je sais que ce n'est sûrement pas toi qui l'as ajouté, mais il faut l'enlever quand même !

Reply:
waouh je crois que j'ai essayé de lancer le debugger et que j'ai fait n'importe quoi ! c'est corrigé merci!